docker/docker-compose.yml

```diff
@@ -50,12 +50,15 @@ services:
     command: npm run start:server
     ports:
       - "1337:1337"
+    working_dir: /cuttle
     volumes: *app-volume
     depends_on:
       - database
 
   client:
     container_name: client
+    # Required to be able to hit http://client:8080 from the Cypress docker container
+    hostname: client
```

Comment:
Why is `hostname` necessary for the client DNS but not for database, which is also used for container to container communication?

Reply:
This is tricky to figure out-- it seems like if the app or service is running INSIDE of docker, it can use these hostnames to talk amongst the other shared clients. When you expose to the host (your local) you can only use the ports and none of the internal hostnames.